### ISSUE TITLE
nix setup?

### ISSUE THREAD

**Issue by nomeata** [kritzcreek/vessel#3]
I’d like to see a `default.nix` here that allows me to run
```
nix-env -iA vessel -f .
```
just like I do with `moc`.

**Comment by kritzcreek** [comment]
Can you do something with nix-shell?
```
$ nix-shell -p cargo rustc
$ cargo build --release
```

I won't be using `nix` to develop this so unless we get a steady stream of contributions from nix users it's going to bitrot (or if there is a way to say "latest nixpkgs" that would maybe work as well?).

**Comment by nomeata** [comment]
Maybe I’ll contribute something. As long as it’s not big and runs in CI it will not bitrot, I assume. For now I fetched the released binary, so not urgent.

**Comment by kritzcreek** [comment]
Fixed in 2f535898c875a81258caacdef2b6fcb1ce9d2dec
